### ISSUE TITLE
Playlist and Playback Issues

### ISSUE THREAD

**Issue by jengajenga** [ffplayout/ffplayout-engine#87]
Hello @jb-alvarado - my issue is long but I will try to be brief

![playplay](https://user-images.githubusercontent.com/65451390/109317342-a9bd9e00-785d-11eb-9af3-513855151a73.png)

1. When I use the settings you see above, the only video that plays is the video i have set for filler.
2. On playlist mode, once the playlist starts, there's no way to stop it. It plays and plays even after reloading the browser, or clicking the stop button or the orange and yellow button. it doesn't respond to commands. And it still only plays the filler.
3. On storage mode, the videos play very well. The only problem is that there is no way to arrange it on the playlist, and the progress bar doesn't show progress.

I hope these are some things you can look at.

Thank you for your time and help.

**Comment by jengajenga** [comment]
@jb-alvarado ... any assistance here brother? thanks.

**Comment by jb-alvarado** [comment]
When the length parameter is not 24 hours, the GUI is not working correctly with the engine. You should also check the logs, I guess it will say why only the filler clip is playing.

**Comment by jengajenga** [comment]
> When the length parameter is not 24 hours, the GUI is not working correctly with the engine. You should also check the logs, I guess it will say why only the filler clip is playing.

Now, the only way this will play properly is when you play it directly from storage. it will reach my rtmp server easily and quickly. it is only that it doesn't make the timer and the progress bar active.

I have done some changes.

Playlist Mode
1. I downloaded playlist generator. I put some media in my folder and generated it with the instructions. it generated the playlist successfully. However, that playlist cannot be modified. If i try to rearrange on the playlist, and click save, a red bar appears and says "refused with error 500". Even the random playlist did not play even though it is 24 hours long.

2. I changed the playlist to 24 hours, it says playlist too short because my videos make 32 minutes. It plays the filler.
3. I changed the playlist length to 32 minutes, it says "Clip less then a second, skip: /opt/tv-media/Tomorrow..mkv" but the video is 20 minutes long. It then opens to play the filler.
4. I removed the filler. It says "no filler set" and plays blank.

Basically, I am unable to get it to play my playlist.

Oh, you could add a delete playlist button?

**Comment by jb-alvarado** [comment]
The playlist generator should run on the same machine then ffplayout-engine is running, because it uses the same ffplayout.yml config. For that reason the length parameter in the config should be set first, and this in hh:mm:ss format (24:00:00).

Try to set playlist length to 24 hours and generate 24 hours playlists.

When you generate a playlist with a different system user, then that user which runs the ffplayout-api, then you cannot edit the playlists in the GUI anymore. You need to change the owner from the generated playlists.

When it say  **Clip less then a second ...** it can mean that you playlist start time is to much in the past and there is no clip which can play at current time.

**Comment by jengajenga** [comment]
> The playlist generator should run on the same machine where the ffplayout-engine is running, because it uses the same **ffplayout.yml** config. For that reason the length parameter in the config should be set first, and this in **hh:mm:ss** format (24:00:00).
> 
> Try to set playlist length to 24 hours and generate 24 hours playlists.
> 
> When you generate a playlist with a different system user, then that user which runs the ffplayout-api, then you cannot edit the playlists in the GUI anymore. You need to change the owner from the generated playlists.
> 
> When it say **Clip less then a second ...** it can mean that you playlist start time is to much in the past and there is no clip which can play at current time.

1. Playlist generated successfully. I also changed user and was able to modify it on the playlist. So that one is ok.

I am opening another thread with a set of other concerns.
